extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaQuarkusConfiguration.java

```diff
@@ -0,0 +1,147 @@
+package io.quarkus.narayana.jta.runtime;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class NarayanaQuarkusConfiguration {
+    // non final properties
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoordinatorEnvironmentBean.commitOnePhase", defaultValue = "true")
```

Comment:
The keys do not follow our configuration guidelines at all. It's definitely not something we want.

Reply:
Okay, in this initial PR I wanted to keep everything as close as possible to what narayana does. Once we have agreed on what we want to be configurable I will add sensible defaults. I asked above if I should be taking the initial set of defaults from what we do on wildfly and then modify based on the specifics of quarkus.